example/v1/index.ts

```diff
@@ -1,8 +1,13 @@
-import {Routing} from '../../src';
+import {RouteMethods, Routing} from '../../src';
 import {getUserEndpoint} from './get-user';
 import {setUserEndpoint} from './set-user';
 
-export const v1Routing: Routing =  {
+export const v1Routing: Routing = {
+  // syntax 1: methods are defined within the endpoint
   getUser: getUserEndpoint,
-  setUser: setUserEndpoint
+
+  // syntax 2: methods are defined within the route
+  setUser: new RouteMethods({
+    post: setUserEndpoint // the Endpoint should have at least the same method specified in .build()
```

Comment:
A route can have multiple handlers (with different I/O schemas) depending on different methods.
It can also be the same Endpoint that handle different methods as well.